app/views/publications/mid_cycle_report/_table_body.html.erb

```diff
@@ -0,0 +1,28 @@
+<thead class="govuk-table__head">
+  <tr class="govuk-table__row">
+    <th scope="col" class="govuk-table__header">Level</th>
+    <th scope="col" class="govuk-table__header">This cycle</th>
+    <th scope="col" class="govuk-table__header">Last cycle</th>
+    <th scope="col" class="govuk-table__header">Percentage change</th>
```

Comment:
Don't think we need to include "Percentage" in the header, as each of the cells have their own percentage sign.

```suggestion
    <th scope="col" class="govuk-table__header">Change</th>
```